templates/zerver/social_auth_select_email.html

```diff
@@ -78,4 +80,15 @@ <h1 class="get-started">Select account</h1>
     </div>
     {% endif %}
 </div>
+
+<script>
+document.addEventListener('DOMContentLoaded', function() {
+    $('.choose-email-box').on('keydown', function(e) {
+        if (e.key === 'Enter' || e.key === ' ') {
+            e.preventDefault();
+            $(this).closest('form').submit();
+        }
+    });
+});
+</script>
```

Comment:
We don't do custom script tags, add code to `web/src/portico/signup.ts` if you need it.